src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, useState } from "react";
+
+import { StyledAccordion } from "./style";
+import { AccordionProps, AccordionSubComponentProps } from "./type";
+import { getChildrenOnDisplayName } from "../../utils/index";
+import { ReactComponent as AccordionPlus } from "../../assets/icon-accordion-plus.svg";
+import { ReactComponent as AccordionMinus } from "../../assets/icon-accordion-minus.svg";
+
+const Accordion = ({ children }: AccordionProps) => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const title = getChildrenOnDisplayName(children, "Title");
+  const body = getChildrenOnDisplayName(children, "Body");
+  return (
+    <StyledAccordion isOpen={isOpen}>
+      <div className="accordion-container">
+        <div
+          className="accordion-title-wrapper"
```

Comment:
follow BEM convention ya:

`.accordion__title-wrapper`

https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/
https://scalablecss.com/naming-bem-wrappers-and-containers/

Reply:
mantapp